deploy/general-deploy/README.md

```diff
@@ -1,30 +1,29 @@
-## Deploy Kubeturbo on Existing Kubernetes Cluster
+## Deploying Kubeturbo
 
-This guide is about how to deploy kubeturbo service in an existing Kubernetes cluster running in a private datacenter. It also applies to a local Kubernetes cluster. Kubeturbo is deployed as mirror on Master nodes.
+Once deployed, the Kubeturbo service enables you to give Turbonomic visibility into a Kubernetes cluster. This cluster can be located in either a private datacenter, or locally. Kubeturbo will be deployed as a mirror on Master nodes.
```

Comment:
Thank you! Please find this fixed in commit 5f761360240441118b8442914927ca20c0a7b739!